janusgraph-core/src/main/java/org/janusgraph/core/schema/DefaultSchemaMaker.java

```diff
@@ -62,6 +62,18 @@ public default PropertyKey makePropertyKey(PropertyKeyMaker factory) {
     }
 
     /**
+     * Creates a new property key with default settings against the provided {@link PropertyKeyMaker} and value.
```

Comment:
Should mention here that the default implementation does not utilize the `Object value`.